homeassistant/components/zha/entity.py

```diff
@@ -107,6 +106,9 @@ def async_update_state_attribute(self, key: str, value: Any) -> None:
     def async_set_state(self, attr_id: int, attr_name: str, value: Any) -> None:
         """Set the entity state."""
 
+    async def async_added_to_hass(self) -> None:
+        """Run when about to be added to hass."""
+
```

Comment:
```suggestion
```

AFAICT its not needed since the one in `Entity` is fine

Reply:
In fact I forgot to remove it.